scripts/tests/Ice/interceptor.py

```diff
@@ -2,4 +2,7 @@
 # Copyright (c) ZeroC, Inc. All rights reserved.
 #
 
-TestSuite(__name__, [ClientTestCase(client=Client(props={"Ice.Warn.Dispatch" : 0}))], libDirs=["interceptortest"])
+if isinstance(Mapping.getByPath(__name__), CSharpMapping):
+    TestSuite(__name__, [ClientServerTestCase(), ClientAMDServerTestCase(), CollocatedTestCase()])
+else:
+    TestSuite(__name__, [ClientTestCase(client=Client(props={"Ice.Warn.Dispatch" : 0}))], libDirs=["interceptortest"])
```

Comment:
Should we have a `TODO` to fix this in the future?

Reply:
sure I will add one